universe/com.goal_smart/manifest.tt

```diff
@@ -48,24 +49,25 @@ class @com.goal_smart
   monitorable list query standings(in req league_id: Entity(com.goal_smart:soccer_league) #_[prompt="what league standings do you want out check out"],
                         out team : Entity(com.goal_smart:soccer_team)
                         #_[canonical={
-                          base=["team", "side", "group"]
+                          base=["team", "side", "group"],
+                          reverse_verb_projection=['played', 'particapted', 'competed']
                         }],
                         out rank: Number
                         #_[canonical = {
-                        base = ["rank", "placement"],
-                        property = ["rankings of #", "# placement", "placement of #"],
+                          base = ["rank", "placement"],
+                          property = ["rankings of #", "# placement", "placement of #"],
                         }],
                         out form: Array(String)
                         #_[canonical = {
-                        base = ["form", "momentum"],
-                        property = ["form of #"]
+                          base = ["form", "momentum"],
+                          property = ["form of #"]
                         }],
                         out points: Number
                         #_[canonical = {
-                        base = ["results", "points"],
-                        property = ["points of #", "# points", "results of #"],
+                          base = ["results", "points"],
+                          property = ["points of #", "# points", "results of #"],
```

Comment:
Btw I think these "property" canonicals are not correct. They construct phrases like:
> find me league standings with points of 55
which are not grammatical.